geomstats/_backend/pytorch/__init__.py

```diff
@@ -755,3 +754,16 @@ def cross(a, b):
 
 def gamma(a):
     return _torch.exp(_gammaln(a))
+
+
+def imag(a):
+    a = _torch.tensor(a)
```

Comment:
check if it is tensor first and only do this if not.

Reply:
Done.